lib/infrataster/server.rb

```diff
@@ -154,10 +154,21 @@ def ssh_config_for_vagrant(name)
 
     def fetch_all_addresses
       result = []
-      ssh_exec('PATH="/sbin:$PATH" ip addr').each_line do |line|
-        #inet 10.0.2.15/24 brd 10.0.2.255 scope global eth0
-        if %r{inet ([^/]+)} =~ line
-          result << $1
+      uname = ssh_exec('uname -s')
+      ssh_exec('UNAME=$( uname -s ); case ${UNAME} in Linux) PATH="/sbin:$PATH" ip addr ;; FreeBSD) /sbin/ifconfig ;; OpenBSD) /sbin/ifconfig ;; esac').each_line do |line|
```

Comment:
I think it is better to check availability of `ip` or `ifconfig` command instead of getting OS.